style.css

```diff
@@ -0,0 +1,102 @@
+/* Generic designing */
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  background-color: #e5e5e5;
+}
+
+/* Header Disigning */
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 20px 0 20px;
+  background-color: #fff;
+}
+
+.logo {
+  height: auto;
+  width: auto;
+  font-family: Poppins, serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #6070ff;
+}
+
+/* Main Disigning */
+
+.main {
+  display: flex;
+  flex-direction: column;
+  background-image: url(/img/header-shapes\ mobile.png);
```

Comment:
- [ ] As requested by the previous CR please 🙏🏽  fix the background image to be similar to the Figma design. Ensure that the changes don't affect the rest of the items, it is not possible to see the background image and the social icons either. 👇🏽 

![image](https://github.com/ali-techworld/Portfolio-Setup/assets/58642949/5d174bf7-113b-4a8a-8ff9-20ede88f8c2d)